src/header/name.rs

```diff
@@ -1727,12 +1163,34 @@ impl HeaderName {
 
     /// Converts a static string to a HTTP header name.
     ///
-    /// This function panics when the static string is a invalid header.
-    ///
     /// This function requires the static string to only contain lowercase
     /// characters, numerals and symbols, as per the HTTP/2.0 specification
     /// and header names internal representation within this library.
     ///
+    /// # Panics
+    ///
+    /// This function panics when the static string is a invalid header.
+    ///
+    /// Until [Allow panicking in constants](https://github.com/rust-lang/rfcs/pull/2345)
+    /// makes its way into stable, the panic message at compile-time is
+    /// going to look cryptic, but should at least point at your header value:
+    ///
+    /// ```text
+    /// error: any use of this value will cause an error
+    ///     --> http/src/header/name.rs:1241:13
+    ///      |
+    /// 1241 |             ([] as [u8; 0])[0]; // Invalid header name
+    ///      |             ^^^^^^^^^^^^^^^^^^
+    ///      |             |
+    ///      |             index out of bounds: the length is 0 but the index is 0
+    ///      |             inside `fastly::http::HeaderName::from_static` at http/src/header/name.rs:1241:13
```

Comment:
did you mean to include `fastly` here?

Reply:
```suggestion
    ///      |             inside `http::HeaderName::from_static` at http/src/header/name.rs:1241:13
```

Reply:
Haha, no, I did not.  I'm so used to seeing it I didn't even notice.  I'm going to force push that change.